pom.xml

```diff
@@ -1,23 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <packaging>pom</packaging>
-    <modules>
-        <module>log-search-utility</module>
-        <module>log-search-tests</module>
-    </modules>
+
     <parent>
         <groupId>uk.gov.justice</groupId>
         <artifactId>maven-parent-pom</artifactId>
         <version>1.3.0</version>
     </parent>
+
     <groupId>uk.gov.justice.log</groupId>
     <artifactId>log-search</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0.0-SNAPSHOT</version>
+
     <properties>
-        <common-bom.version>1.5.1</common-bom.version>
+        <common-bom.version>1.14.0-SNAPSHOT</common-bom.version>
```

Comment:
right now it is in master as a snapshot version and not release ready yet so continuing with snapshot version